src/ucp/proto/proto_common.h

```diff
@@ -140,6 +145,9 @@ typedef struct {
 
     /* Maximum single message length */
     size_t max_frag;
+
+    /* Optimal zero-copy buffer alignment */
+    size_t opt_zcopy_align;
```

Comment:
opt_align
let's remove zcopy terminology from proto_multi align logic, so the name will be shorter and it will also be more generic